willow/app/inputs/rights_statement_input.rb

```diff
@@ -21,35 +21,35 @@ def build_components(attribute_name, value, index, options)
       out << '</div>' # row
 
       # # --- Definition
-      # field = :definition
-      # field_name = name_for(attribute_name, index, field)
-      # field_value = rights_statement.send(field).first
+      field = :definition
+      field_name = name_for(attribute_name, index, field)
+      field_value = rights_statement.send(field).first
 
-      # out << "<div class='row'>"
-      # out << "  <div class='col-md-3'>"
-      # out << template.label_tag(field_name, field.to_s.humanize, required: false)
-      # out << '  </div>'
+      out << "<div class='row'>"
+      out << "  <div class='col-md-3'>"
+      out << template.label_tag(field_name, 'License statement', required: false)
```

Comment:
though not sure... I see dublin core use 'license' http://dublincore.org/documents/dcmi-terms/#terms-license